package.json

```diff
@@ -74,5 +74,9 @@
         "start": "react-scripts start",
         "test": "react-scripts test"
     },
-    "version": "0.0.1"
+    "version": "0.0.1",
+    "resolutions": {
+        "**/@typescript-eslint/eslint-plugin": "^4.1.1",
+        "**/@typescript-eslint/parser": "^4.1.1"
+    }
```

Comment:
Aha, thanks for narrowing down the culprit! I can confirm locally that this fixes the issue. That investigation must have been fun... 😄 

For this particular solution though, I think we can be a little less override-y: `react-scripts` has a `4.0.1` version published that has a dependency on newer `@typescript-eslint/*` versions. Could you switch to bumping the `react-scripts` and `@typescript-eslint/*` packages to newer versions so we don't have to override resolutions in this way?